__tests__/reviews.test.js

```diff
@@ -52,3 +53,49 @@ describe("/api/reviews/:review_id - GET request", () => {
       });
   });
 });
+
+describe("/api/reviews - GET request", () => {
+  it("responds with a status code of 200 and an array of the review objects, with the requested properties and an additional property of comment_count", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((response) => {
+        const { reviews } = response.body;
+        expect(reviews.length).toBe(13);
+        reviews.forEach((review) => {
+          expect(typeof review.comment_count).toBe("number");
+          expect(review).not.toHaveProperty("review.body");
+          expect(typeof review.owner).toBe("string");
+          expect(typeof review.title).toBe("string");
+          expect(typeof review.review_id).toBe("number");
+          expect(typeof review.category).toBe("string");
+          expect(typeof review.review_img_url).toBe("string");
+          expect(typeof review.created_at).toBe("string");
+          expect(typeof review.votes).toBe("number");
+          expect(typeof review.designer).toBe("string");
+          expect(typeof review.title).toBe("string");
+        });
+      });
+  });
+
+  it("the array of review objects should be sorted by descending order of date", () => {
+    return request(app)
+      .get("/api/reviews")
+      .then((response) => {
+        expect([
```

Comment:
Use the response in this expect statement rather than hardcoding response 🙂

Reply:
Now amended